MasterGrant137/ConverSatiation
# 3 Basic docker infrastructure setup

## Pull Request


### Description

Built containers for the postgresql database, flask app, and next app.

### Related Issue

[Setup Docker container infra #3](https://github.com/MasterGrant137/ConverSatiation/issues/3)

### Test Procedure

_Make sure that you have the appropriate `.env` file_
1. In your terminal run `docker-compose build` to build all the containers. (This may take a bit of time)
2. All containers should build without any errors. (If the error is bad enough the container will just exit)
3. Go to `localhost:3000` and you should see the log in form.
